code/game/gamemodes/gang/dominator.dm

```diff
@@ -13,6 +13,7 @@
 	var/datum/gang/gang
 	var/operating = 0	//0=standby or broken, 1=takeover
 	var/warned = 0	//if this device has set off the warning at <3 minutes yet
+	var/spam_prevention = 100
```

Comment:
most defines are in...  the ```__DEFINES``` folder.
The rest are file-specific and can be found at the top and bottom of said files.

Not hard to find to be honest.